rest-tests/src/test/resources/features/operator-deployment.feature

```diff
@@ -7,6 +7,7 @@ Feature: Operator Deployment
   Background:
     Given clean default namespace
       And deploy Syndesis CRD
+    And install cluster resources
```

Comment:
indent

Reply:
hm, IDEA does this all the time when I commit via IDEA :D